test/jasmine/tests/toimage_test.js

```diff
@@ -51,31 +54,51 @@ describe('Plotly.toImage', function() {
     });
 
     it('should throw error with unsupported file type', function(done) {
-        // error should actually come in the svgToImg step
-
-        Plotly.plot(gd, subplotMock.data, subplotMock.layout)
-            .then(function(gd) {
-                Plotly.toImage(gd, {format: 'x'}).catch(function(err) {
-                    expect(err.message).toEqual('Image format is not jpeg, png or svg');
-                    done();
-                });
-            });
+        var fig = Lib.extendDeep({}, subplotMock);
+        var errors = [];
 
+        Plotly.plot(gd, fig.data, fig.layout)
+        .then(function(gd) {
+            try {
+                Plotly.toImage(gd, {format: 'x'});
+            } catch(e) {
+                errors.push(e.message);
+            }
+        })
+        .then(function() {
+            expect(errors.length).toBe(1);
+            expect(errors[0]).toBe('Image format is not jpeg, png, svg or webp.');
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1939/commits/43a615022281b582b0b4b283ef778870f535c54f